docs/docs/glossary/jamstack.md

```diff
@@ -0,0 +1,32 @@
+---
+title: JAMStack
+disableTableOfContents: true
+---
+
+Learn how to use Gatsby to build websites powered by the JAMStack, a modern architecture that uses JavaScript, APIs and markup instead of a databases and server-side programming languages.
+
+## What is the JAMStack?
```

Comment:
I wasn't immediately sure whether this was meant to be h2 or h3. The following paragraphs spend far more time discussing non-JAMStack architecture than describing what the JAMStack is. I think we can keep the non-JAMStack descriptions if we also include some contrast in order to illuminate what the JAMStack is. This could be accomplished by incorporating the first two paragraphs from the next section.